mcstatus/protocol/connection.py

```diff
@@ -1,339 +1,596 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
```

Comment:
```suggestion
```
No need for the `#!` since this is not an executable script.
Python3 files are assumed to be encoded in utf-8. This magic comment is a remnant from Python2 where strings were encoded differently by default.